packages/demo-react-admin/package.json

```diff
@@ -19,6 +19,7 @@
         "lodash": "~4.17.5",
         "prop-types": "^15.7.2",
         "proxy-polyfill": "^0.3.0",
+        "ra-auth-auth0": "*",
```

Comment:
FMI, why did you need this change?

Reply:
This ensure yarn workspace & vite correctly symlink the dependency in dev mode